### PR TITLE
July 9th release notes

### PR DESCRIPTION
**Pull request by qpetraroia**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+### Release notes
+
+* Preview Features
+  * Added support for KEDA v2.11 on Kubernetes version 1.27.
+
+* Bug Fixes
+  * A bug has been fixed that caused issues with Windows Server container pods while using inline volume for 1.24+ clusters.
+
+* Component Updates
+  * Update KEDA addon to v2.10.1 for versions less than Kubernetes version 1.27.
```
Comment (sabbour) [inline CHANGELOG.md:23]: Link to KEDA change log?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+  * Update Azure Monitor for Containers to [v3.1.10](https://github.com/microsoft/Docker-Provider/blob/ci_prod/ReleaseNotes.md).
+  * Hotfixes for Kubernetes images v1.24.9, v1.24.10, v1.25.5, v1.25.6, v1.26.0, v1.26.3, and v1.27.1.
+  * Update microsoft-defender-pod-collector image to 1.0.72.
+  * Update microsoft-defender-security-publisher image to 1.0.64.
```
Comment (sabbour) [inline CHANGELOG.md:27]: Any release notes?

File: CHANGELOG.md (hunk 1 above)
Comment (qpetraroia) [reply CHANGELOG.md:23]: ack

File: CHANGELOG.md (hunk 2 above)
Comment (qpetraroia) [reply CHANGELOG.md:27]: I don't see any change logs for defender

**Review by sabbour — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+  * Added support for KEDA v2.11 on Kubernetes version 1.27.
+
+* Bug Fixes
+  * A bug has been fixed that caused issues with Windows Server container pods while using inline volume for 1.24+ clusters.
+
+* Component Updates
+  * Update KEDA addon to v2.10.1 for versions less than Kubernetes version 1.27.
+  * Update Azure Monitor for Containers to [v3.1.10](https://github.com/microsoft/Docker-Provider/blob/ci_prod/ReleaseNotes.md).
+  * Hotfixes for Kubernetes images v1.24.9, v1.24.10, v1.25.5, v1.25.6, v1.26.0, v1.26.3, and v1.27.1.
+  * Update microsoft-defender-pod-collector image to 1.0.72.
```
Comment (kaarthis) [inline CHANGELOG.md:26]: Do we have link (public docs) for Defender security and Pod collector to show exactly what we are talking about ?

**Review by kaarthis — COMMENTED** [review]
Left one minor nitpick

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,34 @@
...
+
+* Windows 2019 will be retired in Kubernetes v1.33 and above (ETA March 2026). Customers should [upgrade to Windows 2022](https://learn.microsoft.com/azure/aks/upgrade-windows-2019-2022).
+* Kubernetes 1.24 is being deprecated end of July 2023 and support will transition to our [platform support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy). 
+* Starting Kubernetes 1.25, the default cgroups implementation on AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance.
+* A [known issue](https://github.com/Azure/AKS/issues/3718) in Kubernetes version 1.24 is causing name resolution failures in Windows pods. Customers experiencing this issue should upgrade their cluster to Kubernetes version 1.25.
+
+### Release notes
+
+* Preview Features
+  * Added support for [KEDA v2.11](https://github.com/kedacore/keda/blob/main/CHANGELOG.md#v2110) on Kubernetes version 1.27.
```
Comment (shashankbarsin) [inline CHANGELOG.md:17]: this should be under component updates section

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
+
+* Bug Fixes
+  * A bug has been fixed that caused issues with Windows Server container pods while using inline volume for 1.24+ clusters.
```
Comment (shashankbarsin) [inline CHANGELOG.md:20]: what was the issue? do we have a description of it or better a GH issue that we can link to?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,34 @@
...
+
+* Component Updates
+  * Update KEDA addon to [v2.10.1](https://github.com/kedacore/keda/blob/main/CHANGELOG.md#v2101) for versions less than Kubernetes version 1.27.
```
Comment (shashankbarsin) [inline CHANGELOG.md:23]: suggest merging the other KEDA line from above here and covering the status for < 1.27 and > 1.27 in one line here

File: CHANGELOG.md (hunk 3 above)
Comment (qpetraroia) [reply CHANGELOG.md:26]: I don't think we have one, I tried looking and talked to Ahmed about it. @shashankbarsin do you know if we have one?

File: CHANGELOG.md (hunk 5 above)
Comment (qpetraroia) [reply CHANGELOG.md:20]: @shashankbarsin I couldn't find anything in the board, and this was the description I got from the team.

File: CHANGELOG.md (hunk 6 above)
Comment (qpetraroia) [reply CHANGELOG.md:23]: ack

File: CHANGELOG.md (hunk 5 above)
Comment (shashankbarsin) [reply CHANGELOG.md:17]: let's ping Ally and Abel

**Review by kaarthis — APPROVED** [review]

**Pull request merged**